tests/Elements/ElementBlogPostsTest.php

```diff
@@ -29,7 +29,7 @@ public function testGetSummary()
             $object->getSummary(),
             _t(
                 BlogPost::class . 'PLURALS',
-                'A Blog Post|{count} Blog Posts',
+                'A Blog Post|{count} Base Pages',
```

Comment:
Should this read `Blog Posts`?

Reply:
It used to but the Blog module was updated https://github.com/silverstripe/silverstripe-blog/blob/3/lang/en.yml#L45